cloudify_ansible/utils.py

```diff
@@ -262,3 +279,17 @@ def assign_environ(_vars):
 def unassign_environ(_vars):
     for key in _vars.keys():
         del os.environ[key]
+
+
+def get_additional_node_groups(node_name, deployment_id):
+    """This enables users to reuse hosts in multiple groups."""
+    groups = []
+    try:
+        client = get_rest_client()
+    except KeyError:
```

Comment:
Do we really need to catch this error here or we should catch up something else ?